optimism/src/keccak/column.rs

```diff
@@ -16,8 +16,10 @@ use std::ops::{Index, IndexMut};
 
 /// The total number of witness columns used by the Keccak circuit.
 pub const ZKVM_KECCAK_COLS: usize =
-    ZKVM_KECCAK_COLS_CURR + ZKVM_KECCAK_COLS_NEXT + MODE_FLAGS_COLS_LEN + 2;
+    ZKVM_KECCAK_COLS_CURR + ZKVM_KECCAK_COLS_NEXT + MODE_FLAGS_COLS_LEN + STATUS_FLAGS_LEN;
 
+// The number of columns used by the Keccak circuit to represent the status flags.
+const STATUS_FLAGS_LEN: usize = 3;
```

Comment:
Curious that the compiler does not complain that the variable is defined after we use it :thinking: